apps/asap-server/src/controllers/calendars.ts

```diff
@@ -30,21 +32,50 @@ export default class Calendars implements CalendarController {
     this.squidexGraphqlClient = squidexGraphqlClient;
   }
 
-  async fetch(options: {
-    take: number;
-    skip: number;
-  }): Promise<ListCalendarResponse> {
-    const { take = 50, skip = 0 } = options;
-    const { total, items: calendars } =
-      await this.calendarSquidexRestClient.fetch({
-        take,
+  async fetch(): Promise<ListCalendarResponse> {
```

Comment:
When you have basically refactored this code - why have you not moved the business logic (with specific technologies/API behind) into the proper class or file - like move it away from controller?

Reply:
good question! I think most of the logic belongs to this class anyway. The controller wraps stuff like declaring queries and getting the data back, the only thing that is extracted (and was) is the parsing logic.

Mind you - we do not have CONTRIBUTING guide that would actually specify which type of logic should belong to controller and what should be moved up, but there is a ticket for it: https://trello.com/c/yCshrAnG/1666-split-be-controllers-into-2-layers